src/main/java/com/amazonaws/services/dynamodbv2/AmazonDynamoDBLockClient.java

```diff
@@ -553,6 +734,39 @@ private LockItem putLockItemAndStartSessionMonitor(AcquireLockOptions options, S
         return lockItem;
     }
 
+    /**
+     * Builds an updateExpression for all fields in item map and updates the correspoding expression attribute name and
+     * value maps.
+     * @param item Map of Name and AttributeValue to update or create
+     * @param expressionAttributeNames
+     * @param expressionAttributeValues
+     * @return
+     */
+    private String getUpdateExpressionAndUpdateNameValueMaps(Map<String, AttributeValue> item,
```

Comment:
add a github issue to refactor Map<String, AttributeValue> utility methods out of this class.

Reply:
#24 created